tutorials/classical_simulation/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
<ul><li>1st line: I would rather have "This notebook is the first in a series of tutorials on the use of the <code>pulser-simulation</code> module to simulate pulse sequences with noise and errors"</li><li>In 1. SPAM: <code>$\\Ket{g}$</code> instead of <code>$\\ket{g}$</code>; "We model this <strong>by </strong>a probability η</li></ul><p>of <em>not</em> having a given atom during<strong> p</strong>rocessing." or "This is modeled by a probability η of <em>not</em> having a given atom during processing."</p><p>In 2. Laser Noises: "The fluctuations of the properties of the lasers affect their effectiveness in addressing the desired transitions between atomic levels:"; I would also have "Therefore, the laser frequency <strong>they experience/addressed</strong> is shifted by Doppler <strong>shift/shift frequency/is Doppler-shifted</strong> due to thermal motion. This <strong>can be seen  as</strong> a shift in the detuning frequency of the laser, and leads to a <strong>decrease </strong>in the Rydberg population." (we can count them) (does Rydberg population mean something or would "population of atoms in the Rydberg state" be more appropriate ?)</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>